GOVERNANCE.md

```diff
@@ -0,0 +1,90 @@
+# Project Governance
+
+This document defines governance policies for the the projects under the Devfile organization.
+
+- [Maintainers](#maintainers)
+- [Becoming a Maintainer](#becoming-a-maintainer)
+- [Meetings](#meetings)
+- [CNCF Resources](#cncf-resources)
+- [Code of Conduct Enforcement](#code-of-conduct)
+- [Voting](#voting)
+
+## Maintainers
+
+Devfile Maintainers have write access to the [Devile GitHub organization repositories](https://github.com/devfile).
+They can merge their own patches or patches from others. The current maintainers
+can be found in [MAINTAINERS](./MAINTAINERS).  Maintainers collectively manage the project's
+resources and contributors.
+
+This privilege is granted with some expectation of responsibility: maintainers
+are people who care about the Devfile project and want to help it grow and
+improve. A maintainer is not just someone who can make changes, but someone who
+has demonstrated their ability to collaborate with the team, get the most
+knowledgeable people to review code and docs, contribute high-quality code, and
+follow through to fix issues (in code or tests).
+
+A maintainer is a contributor to the project's success and a citizen helping
+the project succeed.
+
+## Becoming a Maintainer
+
+To become a Maintainer you need to demonstrate the following:
+
+  * commitment to the project:
+    * participate in discussions, contributions, code and documentation reviews
+      for 3 months or more,
+    * perform reviews for 5 non-trivial pull requests,
+    * contribute 5 non-trivial pull requests and have them merged,
+  * ability to write quality code and/or documentation,
+  * ability to collaborate with the team,
+  * understanding of how the team works (policies, processes for testing and code review, etc),
+  * understanding of the project's code base and coding and documentation style.
+
+A new Maintainer must be proposed by an existing maintainer by submitting a new post to the
+[Devfile API discussion board](https://github.com/devfile/api/discussions). A simple majority vote of existing Maintainers
+approves the application.
+
+Maintainers who are selected will be granted the necessary GitHub rights,
+and invited to the [private maintainer mailing list](https://lists.cncf.io/g/cncf-devfile-maintainers).
+
+## Meetings
+
+Time zones permitting, Maintainers are expected to participate in the public
+developer meeting, which occurs
+
+Every Tuesday at 9am EST ([video call link](https://meet.google.com/trx-vqbw-ekb))
```

Comment:
Currently, these calls are cancelled once every three weeks due to conflicts in the current attendee's schedules. Should we note this or move the meeting to a more stable time?

Reply:
This is a good point. In general I would like to have a confirmation from non Red Hat maintainers that current call schedule is ok for them and they do not have recurrent conflicts. Then, if we keep current schedule, we need to decide if we should make the every 3 weeks "pause" official and update the calendar invites. Sending an email to the new maintainers mailing list to discuss that.